examples/driving_in_traffic/run.py

```diff
@@ -111,7 +90,24 @@ def _build_scenario():
     os.system(build_scenario)
 
 
-def _wrap_env(env, config):
+def _setup_gpu():
+    gpus = tf.config.list_physical_devices("GPU")
+    if gpus:
+        try:
+            for gpu in gpus:
+                tf.config.experimental.set_memory_growth(gpu, True)
+            logical_gpus = tf.config.list_logical_devices("GPU")
+            print(len(gpus), "Physical GPUs,", len(logical_gpus), "Logical GPUs")
+        except RuntimeError as e:
+            print(e)
```

Comment:
We don't need to raise the error?

Reply:
In this code, we are trying to set the memory growth property of the GPU. However, depending on the user’s GPU setup, we might fail in setting this property. Failure in setting this property does not mean the code is unable to run. Even without the call to `_setup_gpu()`, the code should be able to run using the available GPU or CPU. Therefore, printing a message on the failure to set GPU memory property and allowing the code to continue running appears to be the better step in order not to unnecessarily frustrate the user.

For more reference, see: 
+ https://www.tensorflow.org/guide/gpu#limiting_gpu_memory_growth
+ https://www.tensorflow.org/api_docs/python/tf/config/experimental/set_memory_growth